yarl/__init__.py

```diff
@@ -207,8 +207,10 @@ def build(
         if query and query_string:
             raise ValueError('Only one of "query" or "query_string" should be passed')
         if path is None or query_string is None or fragment is None:
-            raise TypeError('NoneType is illegal for "path", "query_string" and '
-                            '"fragment" args, use string values instead.')
+            raise TypeError(
+                'NoneType is illegal for "path", "query_string" and '
+                '"fragment" args, use string values instead.'
```

Comment:
I usually prefer to have a trailing comma at the end of multiline args list. It makes the style more consistent should the need to add/remove other args arise.
```suggestion
                '"fragment" args, use string values instead.',
```

Reply:
1. This is not multi-args list but the single argument.
2. I don't want to discuss code styling anymore. The code is formatted by `black`, convince them.
3. Trailing comma is a syntax error in Python 3.5, BTW